platform/ios/src/MGLPinAnnotationView.m

```diff
@@ -0,0 +1,72 @@
+#import "MGLPinAnnotationView.h"
+
+@implementation MGLPinAnnotationView
+
+- (instancetype)initWithReuseIdentifier:(nullable NSString *)reuseIdentifier {
+    self = [super initWithReuseIdentifier:reuseIdentifier];
+    _shadowColor = [UIColor blackColor];
+    _pinColor = [UIColor blueColor];
+    _strokeColor = [UIColor whiteColor];
+    _innerColor = [UIColor whiteColor];
```

Comment:
Should setting one of these color properties immediately change the view’s appearance? If so, add a custom setter for each of the properties that forces a redraw.